tests/foreman/cli/test_import.py

```diff
@@ -1338,3 +1411,114 @@ def test_reimport_enable_rh_repos_negative(self, test_data):
                 ssh_import_org[1], files['channels']
             )
         )
+
+    @data(*gen_import_chost_data())
+    def test_import_content_hosts_default(self, test_data):
+        """@test: Import all content hosts from
+        the predefined dataset
+
+        @feature: Import Content-host
+
+        @assert: Profiles for all Content Hosts created
+
+        """
+        tmp_dir = self.default_dataset[0]
+        files = dict(self.default_dataset[1])
+        files['content-views'] = os.path.join(
+            tmp_dir,
+            'exports/CHANNELS/export.csv',
```

Comment:
I see that this path is being used in more than one place, it will be great if you can add a constant for this.